test/oComments.test.js

```diff
@@ -72,12 +76,49 @@ describe("OComments", () => {
 		});
 	});
 
+	describe("when an authentication token is missing", () => {
+		let comments;
+		let rootElement;
+		let scriptElement;
+		let oCommentsReadyListener;
+
+		before((done) => {
+			oCommentsReadyListener = () => {
+				rootElement = comments.oCommentsEl;
+				scriptElement = rootElement.querySelector('script');
+				done();
+			};
+
+			document.addEventListener('oCommentsFailed', oCommentsReadyListener);
+
+			fetchMock.mock('https://comments-auth.ft.com/v1/jwt/', {});
+
+			fixtures.htmlCode();
+			comments = OComments.init('#element');
+		});
+
+		after(() => {
+			document.removeEventListener('oCommentsFailed', oCommentsReadyListener);
+			fetchMock.reset();
+			fixtures.reset();
+		});
+
+		it("won't create a script element", () => {
```

Comment:
I didn't add a "bad" test for "set source to `embed.js`" or "set an `onload` attribute" because they rely on `scriptElement` existing, which it does not. The error that is spit out is `cannot read null of src`. 

I don't see the point in having these tests but that could be my end-of-the-day brain talking.